scripts/scil_compute_NODDI.py

```diff
@@ -0,0 +1,185 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute NODDI [1] maps using AMICO.
+"""
+
+import argparse
+import os
+import shutil
+import tempfile
+
+import amico
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+amico.core.setup()
+
+EPILOG = """
+Reference:
+    [1] Zhang H, Schneider T, Wheeler-Kingshott CA, Alexander DC.
+        NODDI: practical in vivo neurite orientation dispersion
+        and density imaging of the human brain.
+        NeuroImage. 2012 Jul 16;61:1000-16.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, epilog=EPILOG,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    p.add_argument('in_dwi',
+                   help='DWI file acquired with a NODDI compatible protocol')
+
+    p.add_argument('--mask',
+                   help='Mask filename')
+
+    g = p.add_argument_group('Gradients / scheme')
+    g.add_argument('--bval',
+                   help='Bval filename, in FSL format')
+    g.add_argument('--bvec',
+                   help='Bvec filename, in FSL format')
+    g.add_argument('--scheme_file',
+                   help='AMICO scheme file '
+                        'can replace --bval/--bvec.')
+    g.add_argument('--bstep', type=int, nargs='+',
+                   help='List of unique bvals in your data. It prevents errors'
+                        ' when each bval is around the actual bval')
```

Comment:
Why not add a "tolerance" bvalues like we have in other scripts and like I asked Francois to add to the COMMIT scriipt

Reply:
I prefer that option as well